tests/js_tests.rs

```diff
@@ -6,17 +6,34 @@ use wasm_bindgen::JsValue;
 
 pub use fluvio_client_wasm::*;
 
-#[wasm_bindgen_test]
-async fn simple() {
+async fn get_fluvio() -> Fluvio {
     let fluvio = Fluvio::connect("ws://localhost:3000".into()).await;
     assert!(fluvio.is_ok());
-    let fluvio = fluvio.unwrap();
-    let ret = simple_js(fluvio, Offset::from_end(1)).await;
+    fluvio.unwrap()
+}
+
+#[wasm_bindgen_test]
+async fn simple() {
+    let ret = simple::setup(get_fluvio().await).await;
```

Comment:
Can you call `get_fluvio()` just once? Either pass `&Fluvio` or clone `Fluvio` to pass?

Reply:
Yeah, I tried this. You can't pass in `&Fluvio` through wasm-bindgen and `Fluvio` doesn't derive `Clone`.